src/python/pants/backend/codegen/grpcio/python/grpcio_run.py

```diff
@@ -45,6 +47,13 @@ def execute_codegen(self, target, target_workdir):
       if exit_code != 0:
         raise TaskError('{} ... exited non-zero ({}).'.format(cmdline, exit_code),
                         exit_code=exit_code)
+      # Create __init__.py in each subdirectory of the target directory so that setup_py recognizes
+      # them as modules.  Note that we do not use pex_build_util.py's identify_missing_init_files()
+      # because it expects a set of all the filenames and we do not want to precompute that, as we
+      # only have the root directory name at this point.
```

Comment:
Hm. It feels like it would still be good to use that method, as it is easier that having this comment explaining why we're not using that method.

Reply:
Agreed. I think this will work:

```python
source_files = [p.as_posix() for p in Path(target_workdir).rglob("*.py")]
for missing_init in identify_missing_init_files(source_files):
  missing_init.touch()
```

The `as_posix` is to convert from `pathlib.Path -> str`.

Reply:
I updated the PR. Please take a look. Because `identify_missing_init_files()` uses `os.sep` to split the pathnames into components, I used `str()` instead of `as_posix()`.

Reply:
Ah cool! I didn't realize they implement `str()` to do that (return the native representation). That's great and much better. Thanks!